test/e2e/utils.go

```diff
@@ -31,8 +31,9 @@ import (
 	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
 )
 
-const ReadTimeout = 120 * time.Second
+const ReadTimeout = 180 * time.Second
```

Comment:
Why is this change needed? There should be no such difference compared to the original code with configuration in NNS. (I was doing shotgun debugging in my PRs, if you saw it there)

Reply:
I have reuse the condition check and it was 180 there so I have put it for all 

https://github.com/nmstate/kubernetes-nmstate/pull/261/files#diff-27d7aee9b3a12c999081eedcb5a054aeL124